osf_tests/factories.py

```diff
@@ -405,7 +406,7 @@ def _create(cls, target_class, project=None, is_public=False,
         project.save()
 
         # Default registration parameters
-        schema = schema or get_default_metaschema()
+        schema = schema or get_default_test_schema()
```

Comment:
Don't make this change if it's the source of legacy test breakages. We can have a best practice of manually specifying default_test_schema until we have time to clean up any old stuff.